ui/app/src/components/GlobalApp/GlobalApp.tsx

```diff
@@ -52,11 +60,79 @@ interface GlobalAppProps {
   footerHtml: string;
 }
 
+export function GlobalAppRouterProvider(props: GlobalAppProps) {
+  const { passwordRequirementsMinComplexity } = props;
+  const globalNavigation = useGlobalNavigation();
+  const { classes } = useStyles();
+
+  const router = createHashRouter(
+    createRoutesFromElements(
+      <Route path="/" element={<GlobalApp {...props} />}>
+        <Route path="/sites" element={<SiteManagement />} />
```

Comment:
Leave this route for _backward compat_, but let's modify the main route to be "/projects"